src/index.js

```diff
@@ -2,6 +2,10 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import axios from 'axios';
+import "semantic-ui-css/semantic.min.css"
+
+axios.defaults.baseURL = "http://localhost:3000/api/"
```

Comment:
have you deployed your api? in that case please make sure that you add the right URL

Reply:
done